tasmota/include/tasmota_template_legacy.h

```diff
@@ -471,7 +471,9 @@ const uint16_t kGpioConvert[] PROGMEM = {
   AGPIO(GPIO_IEM3000_TX),     // IEM3000 Serial interface
   AGPIO(GPIO_IEM3000_RX),     // IEM3000 Serial interface
   AGPIO(GPIO_ZIGBEE_RST),     // Zigbee reset
-  AGPIO(GPIO_DYP_RX)
+  AGPIO(GPIO_DYP_RX),
```

Comment:
No changes allowed in this file. Pls remove.

Reply:
Still not resolved

Reply:
See https://github.com/arendst/Tasmota/pull/16014